edx_sailthru/sailthru_content/services/sailthru_translation_service.py

```diff
@@ -46,7 +46,8 @@ def _create_course_vars(self, course, course_run, url, site_name, program_dictio
             'course_run': True,
             'marketing_url': url,
             'course_id': course['key'],
-            'course_run_id': course_run['key']
+            'course_run_id': course_run['key'],
+            'marketable': 1,  # Mark the course_run to be marketable when updated through this script
```

Comment:
No reason really. I chose `1` because we already have this pattern for variable `activated` on the user object we sync with Sailthru